tests/conftest.py

```diff
@@ -2,47 +2,65 @@
 import os
 
 import pytest
+from git import Repo
 
 import astrodb_utils
-from astrodb_utils import load_astrodb
+from astrodb_utils import build_db_from_json, check_database_settings
 from astrodb_utils.publications import ingest_publication
 
 logger = logging.getLogger(__name__)
 
-# TODO: Figure out how to import ReferenceTables
-REFERENCE_TABLES = [
-    "Publications",
-    "Telescopes",
-    "Instruments",
-    "PhotometryFilters",
-    "Versions",
-    "RegimeList",
-    "AssociationList",
-    "SourceTypeList",
-    "ParameterList",
-    "CompanionList",
-]
-
-
-DB_NAME = "tests/test-template-db.sqlite"
-DB_PATH = "astrodb-template-db/data"
-SCHEMA_PATH = "astrodb-template-db/schema/schema.yaml"
-CONNECTION_STRING = "sqlite:///" + DB_NAME
+# Make sure the astrodb-template-db repository is cloned and updated
+### BROKEN. - doesn't work with improve_load_astrodb branch - git pull manually
+###
+template_schema_path = os.path.join("tests", "astrodb-template-db")
+branch = "improve_load_astrodb"  #"main"
+logger.info(f"Checking out branch '{branch}' of the astrodb-template-db repository.")
+if os.path.exists(template_schema_path):
+    template_repo = Repo(template_schema_path)
```

Comment:
This fails for me. I thought the point was that the github action would take care of pulling the template repo, not the python code.

Reply:
This was trying to make sure it also works locally.  This doesn't work so that's why there's a note about git pulling manually.